modules/dlm/src/main/java/org/elasticsearch/dlm/DataLifecycleService.java

```diff
@@ -371,8 +371,8 @@ private void updatePollInterval(TimeValue newInterval) {
         maybeScheduleJob();
     }
 
-    private void updateRolloverConditions(RolloverConditions newRolloverConditions) {
-        this.rolloverConditions = newRolloverConditions;
+    private void updateRolloverConditions(RolloverConfiguration newRolloverConfiguration) {
```

Comment:
Should we rename this method now?

Reply:
thank you for pointing it out, I missed it.